module4/homework/appendNewRecipe/AppendNewRecipe.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <deque>
+#include <list>
+#include <sstream>
+#include <string>
+#include <vector>
+
+bool AppendNewRecipe(std::vector<std::string> steps,
+                     const std::list<std::string>& ingredients,
+                     const std::deque<std::pair<size_t, char>>& amount);
+
```

Comment:
I think it's quite unimportant but I have to ask you .It is good idea to put there these function declarations? (I mean FormatIngrediebts and FormatRecipit ). You only use them in cpp file. What do you think about it?

Reply:
But they are used in test.cpp file

Reply:
Ohh OK. Sorry about it. So everything is great!